charm/src/charm.py

```diff
@@ -56,7 +72,6 @@ def __init__(self, *args):
     @property
     def version(self) -> str:
         """Report the current version of the app"""
-        # TODO: get the version somehow and return it
```

Comment:
Isn't it still TODO?

Reply:
It is, but pylint doesn't like that and will block it. I could tell it to ignore it, but I'm not sure that's better than just leaving out the comment

Reply:
Okay, we can remove it since the “?” version indicates that we need to change it